pyfr/backends/openmp/generator.py

```diff
@@ -7,21 +7,42 @@ class OpenMPKernelGenerator(BaseKernelGenerator):
     def render(self):
         if self.ndim == 1:
             inner = '''
-                    for (int _xi = cb; _xi < cb + nci; _xi += SOA_SZ)
+                    for (int _xi = 0; _xi < SZ; _xi += SOA_SZ)
+                    {{
+                        #pragma omp simd
+                        for (int _xj = 0; _xj < SOA_SZ; _xj++)
+                        {{
+                            {body}
+                        }}
+                    }}'''.format(body=self.body)
+            outer = '''
```

Comment:
Not sure if inner/outer are the best names?  One looks like a clean up kernel?

Reply:
How about core/clean?

Reply:
Fine.